tests/workflow_utils/test_finish_website.py

```diff
@@ -41,7 +43,16 @@ def test_write_workflow_status(monkeypatch, tmp_path):
     monkeypatch.setenv("CYLC_WORKFLOW_SHARE_DIR", str(tmp_path))
     finish_website.update_workflow_status()
     with open(web_dir / "status.html", "rt", encoding="UTF-8") as fp:
-        assert fp.read() == "<p>Finished</p>\n"
+        content = fp.read().splitlines()
+        # Check finish status.
+        assert content[0] == "<p>Finished</p>"
+        # Check dynamic finish version added.
+        cset_version = version("CSET")
+        assert content[1] == "<p>CSET v" + cset_version + "</p>"
+        # Check dynamic date created added and valid datetime
+        assert "<p>Date completed: " in content[2]
+        datestr = datetime.datetime.strptime(content[2][19:38], "%Y-%m-%d %H:%M:%S")
+        assert isinstance(datestr, datetime.datetime)
```

Comment:
```suggestion
        content = fp.read()
    # Check status is written correctly.
    pattern = r"<p>Completed at \d{4}-\d\d-\d\d \d\d:\d\d using CSET v.+</p>\n"
    assert re.fullmatch(pattern, content)
```
You'll need to `import re` at the top of the file.